github-backup.py

```diff
@@ -8,15 +8,30 @@
 """
```

Comment:
Feel free to add yourself to the author list..

Reply:
Done.